### ISSUE TITLE
No loaded mesh when importing URDF

### ISSUE THREAD

**Issue by danielemorra98** [RobotecAI/o3de-ros2-gem#282]
**Bug description**
When I load the URDF model, the meshes related to the <visual> tags are not loaded. The path of my files are the same as it is described [here](https://github.com/RobotecAI/o3de-ros2-gem/issues/165#issuecomment-1277287346), and I do not get any errors or warnings from the _Asset Processor logs_ . 

**Test environment**
 - OS: Ubuntu 20.04

**Comment by adamdbrw** [comment]
since paths are changed to relative by the importer, they need to be correct after such change. Make sure your urdf and meshes directory follow this structure.

**Comment by danielemorra98** [comment]
I think that the AssetProcessor cannot load .dae mesh files; when I tried with .fbx files it went rightly.

**Comment by adamdbrw** [comment]
You need to enable it through a setting. There could be problems with the scale unless our PR was already merged.
Registry/sceneassetimporter.setreg in the Project should be:
```
{
	"O3DE":
	{
		"SceneAPI":
		{
			"AssetImporter":
			{
				"SupportedFileTypeExtensions":
				[
					".dae",
					".fbx",
					".stl",
					".gltf",
					".glb"
				]
			},
			"MaterialConverter": 
			{
				"Enable": true,
				"DefaultMaterial": "Materials/Presets/PBR/default_grid.material"
			},
			"TangentGenerateComponent":
			{
				"DefaultGenerationMethod": "FromSourceScene",
				"DebugBitangentFlip": false
			},
			"SkinRule":
			{
				"DefaultMaxSkinInfluencesPerVertex": 8,
				"DefaultWeightThreshold": 0.001
			},
			"ModelBuilder": 
			{
				// When false, scenes with multiple meshes assigned to the same material but with different vertex layouts will successfully process
				// When true, stricter error checking will cause the asset to fail to process with an error message indicating why
				"MismatchedVertexLayoutsAreErrors": false 
			}
		}
	}
}

```

**Comment by danielemorra98** [comment]
I tried with these settings and I obtained the following WARNING msgs:
![Screenshot from 2022-10-17 10-34-58](https://user-images.githubusercontent.com/48955695/196130335-5c45b352-a1f9-435c-8130-370b5f4433e3.png)
Unfortunately still no meshes loaded and I don't think it is a problem of the meshes since Blender loads them correctly.

**Comment by adamdbrw** [comment]
@danielemorra98 the meshes are likely loaded but 1/100th of size. I mentioned the scale patch. The UV maps are a separate issue being worked on in O3DE AFAIK.

**Comment by michalpelka** [comment]
Closing - this issue was resolved with the introduction of #297.
